# Inputs
string table_name
string entry_name
float64 value
---
# Outputs
bool success